pkg/networkservice/common/heal/server.go

```diff
@@ -112,6 +112,7 @@ func (f *healServer) Request(ctx context.Context, request *networkservice.Networ
 		defer cw.mut.Unlock()
 
 		if cw.cancel != nil {
+			log.FromContext(ctx).Debug("canceling previous heal")
```

Comment:
Please also add here some fields to make easier looking for heal log messages.

Reply:
Done